products/opensuse/main.pm

```diff
@@ -75,6 +76,8 @@ testapi::set_distribution(DistributionProvider->provide());
 # Set serial failures
 $testapi::distri->set_expected_serial_failures(create_list_of_serial_failures());
 
+return 1 if load_yaml_schedule;
```

Comment:
I guess it's too early to call it here, should be after we set all the variables.